### PR TITLE
spektrum-scanner: fix plot_rssi.py script

### PR DESCRIPTION
**Pull request by smlng**
Adapt python script to comply with python coding conventions, i.e.
fixing error reported by flake8.

### CODE REVIEW

File: spectrum-scanner/tools/plot_rssi.py [hunk 1]
```diff
@@ -65,7 +66,7 @@ def plot_rssi(self, port):
                     tidx = int(timestamp / (self.dt * 1000000)) % (Z.shape[1])
                 except ValueError:
                     continue
-                logging.debug("data: tidx=%d if=%d t=%d", tidx, iface_id, timestamp)
+                logging.debug("data: tidx={} if={} cnt={} t={}".format(tidx, iface_id, count, timestamp))
```
Comment (gebart) [inline spectrum-scanner/tools/plot_rssi.py]: Why is this change necessary?
The original statement avoids evaluating the format when logging is disabled, an insignificant performance difference here, but in other situations the impact can be noticeable.

Comment (gebart) [reply spectrum-scanner/tools/plot_rssi.py]: Like I wrote, it does not make any difference here, but in general, it would be undesirable to evaluate the format string at all if it is going to be discarded anyway because of loglevel < LOG_DEBUG. To reduce the developer confusion and to adhere to best practices, I would prefer to stay with the recommended style of pushing the variables as arguments instead of preformatting the message.

Some relevant articles: 
https://reinout.vanrees.org/weblog/2015/06/05/logging-formatting.html
https://www.reddit.com/r/Python/comments/387oog/when_will_logging_use_newstyle_string_formatting/
https://realpython.com/python-logging/#logging-variable-data

Comment (smlng) [reply spectrum-scanner/tools/plot_rssi.py]: mhm, I see ...

**Comment by smlng** [comment]
@gebart addressed comments

**Comment by smlng** [comment]
I tested 😉
